cmd/acmesolver/app/app.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2020 The Jetstack cert-manager contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"context"
+
+	"github.com/jetstack/cert-manager/pkg/issuer/acme/http/solver"
+	"github.com/spf13/cobra"
+)
+
+func NewACMESolverCommand(ctx context.Context) *cobra.Command {
```

Comment:
For consistency between all our `cmd` packages, can we make this accept a `stopCh` so the initialisation and exit code can be fairly similar between all of them? Don't need to make that set up logic reusable or anything, but it makes it easier to read our entry points and ensure they are correct if they are all similar 😄